MidasCivil_Adapter/PrivateHelpers/MetaData.cs

```diff
@@ -0,0 +1,273 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2021, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.IO;
+using System.Collections.Generic;
+using System.Text;
+using BH.oM.Adapters.MidasCivil;
+using BH.Engine;
+using System;
+
+namespace BH.Adapter.MidasCivil
+{
+    public partial class MidasCivilAdapter
+    {
+        private bool SetMetaData(MetaData MidasMetaData)
```

Comment:
As discussed this would be a `Create` method within the Adapter - see `BarReleases` for a template: https://github.com/BHoM/MidasCivil_Toolkit/blob/master/MidasCivil_Adapter/CRUD/Create/Properties/BarReleases.cs

You will need to add an `if` statement to `Create.cs`.

Reply:
1. Add if statement to `Create`.cs: https://github.com/BHoM/MidasCivil_Toolkit/blob/master/MidasCivil_Adapter/CRUD/Create/Create.cs
2. Add `CreateCollection` method for `Metadata` see `BarRelease` example: https://github.com/BHoM/MidasCivil_Toolkit/blob/master/MidasCivil_Adapter/CRUD/Create/Properties/BarReleases.cs
3. Add `FromMetadata` method to `Convert` class ToMidasCivil see `BarRelease` example: https://github.com/BHoM/MidasCivil_Toolkit/blob/master/MidasCivil_Adapter/Convert/ToMidasCivil/Properties/FromBarRelease.cs

Similar for Read, but it's `Read.cs`, `ReadMetaData` and `Convert.ToMetadata`